### PR TITLE
Add new cs for report tests

### LINKED ISSUE #4473: Add new .cs file for ReportScreen tests
Add a new file for all ReportScreen tests so they can be grouped

### PR DESCRIPTION
**Pull request by ApocalypticFish**
## What's this?
Adds a cs file to contain all of the report UI tests

### Relationships
#4473 

Closes #4473 

## Why do we want this?
Better categorisation of UI tests

## How is it done?
It adds a cs file for tests to go in

### Why not another way?
https://www.thebeaverton.com/2018/07/ceo-of-huawei-refuses-to-stop-using-expression-its-my-way-or-the-huawei/

## Review hints
There should be a new CS file

## :squid: Permissions
Anyone whomstever would like to

### CODE REVIEW

File: Toggl.Tests.UI/ReportsTests.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using NUnit.Framework;
+using Xamarin.UITest;
+using static Toggl.Tests.UI.Extensions.MainExtensions;
+using static Toggl.Tests.UI.Extensions.StartTimeEntryExtensions;
+namespace Toggl.Tests.UI
+{
+    public class ReportsTests
```
Comment (arturs105) [inline Toggl.Tests.UI/ReportsTests.cs]: This needs to be sealed

File: Toggl.Tests.UI/Toggl.Tests.UI.projitems [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
+    <Compile Include="$(MSBuildThisFileDirectory)Extensions\SignUpExtensions.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)SignUpTests.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)ForgotPasswordTests.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)ReportsTests.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)EditTimeEntryTests.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="$(MSBuildThisFileDirectory)Helpers\" />
+    <Folder Include="$(MSBuildThisFileDirectory)Extensions\" />
+  </ItemGroup>
```
Comment (amulware) [inline Toggl.Tests.UI/Toggl.Tests.UI.projitems]: (Almost!) exclusively whitespace change. Please undo/redo without the line ending changes.

File: Toggl.Tests.UI/ReportsTests.cs (hunk 1 above)
Comment (kornelijepetak) [reply Toggl.Tests.UI/ReportsTests.cs]: Interestingly enough, I have forgotten to comment that same thing on the EditTE UI test cs file you merged yesterday. Feel free to do another PR.

File: Toggl.Tests.UI/Toggl.Tests.UI.projitems (hunk 2 above)
Comment (ApocalypticFish) [reply Toggl.Tests.UI/Toggl.Tests.UI.projitems]: Reverted this commit.

**Comment by arturs105** [comment]
🤔 Something fishy is going on here, since it shows "0 files changed".

**Comment by ApocalypticFish** [comment]
I reverted the changes as it kept making whitespace changes. I've committed them again a second ago as it was only showing the 1 line changed in 
Toggl.Tests.UI/Toggl.Tests.UI.projitems whereas previously it was changing the spacing in the whole thing.

File: Toggl.Tests.UI/ReportsTests.cs (hunk 1 above)
Comment (ApocalypticFish) [reply Toggl.Tests.UI/ReportsTests.cs]: Hey, is this sealed now?

File: Toggl.Tests.UI/ReportTests.cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+﻿using NUnit.Framework;
+
+namespace Toggl.Tests.UI
+{
+    [TestFixture]
+    public sealed class ReportTests
```
Comment (arturs105) [inline Toggl.Tests.UI/ReportTests.cs]: I think this should be `ReportsTests`

Comment (ApocalypticFish) [reply Toggl.Tests.UI/ReportTests.cs]: Damn, yep will update

**Pull request merged**